.eslintrc

```diff
@@ -51,6 +17,5 @@
     "jest": true,
     "socket": true,
     "webpackIsomorphicTools": true
-  },
-  "plugins": ["prettier", "react-hooks"]
```

Comment:
We need prettier plugin, it works together with the prettier config and the plugin in IDEs to match the rules everywhere.

Reply:
Same as above. CRA does not have it, so I removed. Re-added it though.